<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppSelector</class>
 <widget class="QDialog" name="AppSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Application</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>390</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
  </widget>
  <widget class="QListView" name="listInstalledApps">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>571</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label0">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>151</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Installed applications:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>335</y>
     <width>58</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Filter:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="filterBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>330</y>
     <width>531</width>
     <height>32</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Application Name</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AppSelector</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AppSelector</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
